web/src/engine/websites/GourmetScans.ts

```diff
@@ -5,25 +5,37 @@ import * as Madara from './decorators/WordPressMadara';
 import * as Common from './decorators/Common';
 
 const pageScript = `
-    new Promise((resolve, reject) => {
-        try {
-            let t = new RocketLazyLoadScripts;
-            t._loadEverythingNow();
-        } catch (error) {}
+    new Promise( (resolve, reject) => {
+        let tries = 0;
+        const interval = setInterval(function () {
+            try {
+                if (CryptoJS) {
+                    clearInterval(interval);
+                    let imgdata = JSON.parse(CryptoJS.AES.decrypt(chapter_data, wpmangaprotectornonce, {
+                        format: CryptoJSAesJson
+                    }).toString(CryptoJS.enc.Utf8));
+                    resolve(JSON.parse(imgdata));
+                }
+            } catch (error) {
+                clearInterval(interval);
+                reject(error);
+            } finally {
+                tries++;
+                if (tries > 10) {
```

Comment:
Seems to me like the abort condition is intended to be 10 seconds instead of 10 attempts.
Either use `FetchWindowScript()` with timeout of 10 seconds, or use time as condition:
```javascript
const start = Date.now();
// ...
if(Date.now() - start > 10_000) {
    reject();
}
```

Reply:
Turns out PagesSinglePageJS doesnt use timeout parameter. 
Gotta use first solution then .